app/views/insured/family_members/_family_members.html.erb

```diff
@@ -27,7 +27,7 @@
       <label class="static_label label-floatlabel" for="relation">RELATIONSHIP</label>
       <span class="field_value floatlabel form-control active-floatlabel" id="relation"><%= l10n(".self").to_s.upcase %></span>
       <%= link_to personal_insured_families_path, remote: true, class: 'close close-2' do %>
-        <i class="fas fa-pencil-alt"></i>
+        <i class="fas fa-pencil-alt"><span class="hide">Edit</span></i></i>
```

Comment:
should this 'edit' be in an l10n?

Reply:
Good catch, changed here: https://github.com/ideacrew/enroll/pull/3459/commits/f2207fd6d20777755e79e7918d3773a203d74805